addons/unattended-upgrades/README.md

```diff
@@ -0,0 +1,3 @@
+# Unnatended Upgrades
+
+This addon will automate upgrading system packages of the distro of your choice.
```

Comment:
We should write a bit more about this addon. I'd expect at least:

* Information that KubeOne 1.3+ is required to use this addon (because of the node labels)
* What's included in the addon (scripts for apt/yum, FLUO, kured)
* Deployment instructions
  * Document that we use node labels, so you can just deploy everything and only the needed manifest will be deployed
* Information about permissions
  * This addon requires a lot of permissions, we should note that. I noticed at least privileged, run as root, hostPID.
* Information about is there anything that should be configured before deploying the addon